command_test.go

```diff
@@ -2921,3 +2921,29 @@ func TestUnknownFlagShouldReturnSameErrorRegardlessOfArgPosition(t *testing.T) {
 		})
 	}
 }
+
+func TestHelpFuncExecuted(t *testing.T) {
+	helpExecuted := false
+
+	child := &Command{Use: "child", Long: "Long description", Run: emptyRun}
+	child.SetHelpFunc(func(cmd *Command, args []string) {
+		helpExecuted = true
+
+		if cmd.Context() == nil {
+			t.Error("Context is nil")
+		}
+	})
+
+	rootCmd := &Command{Use: "root", Run: emptyRun}
+	rootCmd.AddCommand(child)
+
+	output, err := executeCommand(rootCmd, "help", "child")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+
+	checkStringContains(t, output, rootCmd.Long)
```

Comment:
I don't get this check here...
`rootCmd.Long` is empty; I assume you meant to use `child.Long`?
But then the `output` is also empty because the `HelpFunc` does not actually print any help.  I assume it should, so you can verify it?  Then you don't need to do the `helpExecuted` check.

Reply:
Ah copy paste without full understanding :) I'll fix it up to do the print